flyte-single-binary-local.yaml

```diff
@@ -41,9 +41,11 @@ tasks:
       - container
       - sidecar
       - K8S-ARRAY
+      - ray
```

Comment:
Could we remove it? Otherwise, people always have to install Kuberay in the sandbox.

Reply:
Thanks for the catch. Reverted it back